pytorch_lightning/utilities/enums.py

```diff
@@ -111,16 +111,16 @@ def is_interactive_compatible(self) -> bool:
     DDP_FULLY_SHARDED = "ddp_fully_sharded"
 
 
-class DeviceType(LightningEnum):
```

Comment:
Can we deprecate this rather than delete? See example of Enum which prints a deprecation message from Flash: https://github.com/PyTorchLightning/lightning-flash/blob/79b943f858417717b926dcf4bd4d3e2be0788a1f/flash/core/data/data_source.py#L21

Reply:
This is cool! https://github.com/PyTorchLightning/lightning-flash/blob/79b943f858417717b926dcf4bd4d3e2be0788a1f/flash/core/utilities/on_access_enum_meta.py#L4

Reply:
Added in #10505

Reply:
Dope !